app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+    # ページごとの完全なタイトルを返します。 # コメント行
```

Comment:
細かいですが、2文字余分なスペースが頭に入ってそうです
```suggestion
  # ページごとの完全なタイトルを返します。 # コメント行
```

Reply:
対応しました。